### PR TITLE
New available embed

### PR DESCRIPTION
**Pull request by robertt**
This PR introduces a new available embed, which is a lot more obvious and eye-catching, along with more information included.

image

### CODE REVIEW

**Review by alii — APPROVED** [review]

**Review by driaug — APPROVED** [review]
Looks fine to me!

File: src/modules/helpchan.ts [hunk 1]
```diff
@@ -33,16 +33,15 @@ export class HelpChanModule extends Module {
 	CHANNEL_PREFIX = 'help-';
 
 	AVAILABLE_EMBED = new MessageEmbed()
-		.setColor(TS_BLUE)
+		.setColor('#77b155')
```
Comment (ronthecookie) [inline src/modules/helpchan.ts:36]: Add a const for this color [here](https://github.com/typescript-community/community-bot/blob/master/src/env.ts#L35)?

File: src/modules/helpchan.ts [hunk 2]
```diff
@@ -33,16 +33,15 @@ export class HelpChanModule extends Module {
...
 		.setDescription(
-			'This help channel is now **available**, which means that ' +
-				'you can claim it by typing your question into it. ' +
-				'Once claimed, the channel will move into the **Help: Ongoing** category, and ' +
-				`will be yours until it has been inactive for ${
-					dormantChannelTimeout / 60 / 60
-				} hours or is closed ` +
-				'manually with `!close`. When that happens, it will be set to **dormant** and moved into the **Help: Dormant** category.\n\n' +
-				"Try to write the best question you can by providing a detailed description and telling us what you've tried already.",
+			'✅ **Send your question here to claim the channel**\n' +
```
Comment (ronthecookie) [inline src/modules/helpchan.ts:38]: Replacing with a template string would make it easier for future people to edit.

**Review by ronthecookie — CHANGES_REQUESTED** [review]

File: src/modules/helpchan.ts (hunk 1 above)
Comment (robertt) [reply src/modules/helpchan.ts]: Done

File: src/modules/helpchan.ts (hunk 2 above)
Comment (robertt) [reply src/modules/helpchan.ts]: Done

**Review by ronthecookie — APPROVED** [review]
Doing two entirely separate things in one PR is bad but oh well.

**Pull request merged**